code-of-conduct.md

```diff
@@ -74,8 +74,7 @@ CHAOSS members or the general public. We will not name victims without their
 affirmative consent.
 
 The CHAOSS Code of Conduct Team will report on a yearly basis the number of
-incidence reports. The CHAOSS Code of Conduct Team shall consist of three members 
-elected by the community every two years or when needed.
+incidence reports. The CHAOSS Code of Conduct Team shall consist of three members elected by the community every two years or when needed.
```

Comment:
This is better in two lines, as it was before this patch (that helps to readability of the unformatted text.

Reply:
Ok. I will revert this change back.